src/core/cgroup.c

```diff
@@ -4667,3 +4707,11 @@ static const char* const cgroup_memory_accounting_metric_table[_CGROUP_MEMORY_AC
 };
 
 DEFINE_STRING_TABLE_LOOKUP(cgroup_memory_accounting_metric, CGroupMemoryAccountingMetric);
+
+static const char *const cgroup_limit_type_table[_CGROUP_LIMIT_TYPE_MAX] = {
+        [CGROUP_LIMIT_MEMORY_MAX]  = "EffectiveMemoryMax",
+        [CGROUP_LIMIT_MEMORY_HIGH] = "EffectiveMemoryHigh",
+        [CGROUP_LIMIT_TASKS_MAX]   = "EffectiveTasksMax",
```

Comment:
so, the strings here do not match the enum names (the strings carry the word "effective" which the enum names do not). Which is OK, if the strings are more "specific" and the enum is kept more "generic", I guess. But if so, then the name of the table array should somehow make this clear, that this is not a generic string table for CGroupLimitType, but a specific one for the "effective" case. Or in other words, please rename the table (and the to_string/from_string() helpers) to cgroup_limit_type_effective_table[] or something like that. or cgroup_effective_limit_type_table[], or so.

Reply:
Moved over to https://github.com/systemd/systemd/pull/31046.